src/components/charts/PercentageStackedBarChart/PercentageStackedBarChart.less

```diff
@@ -1,23 +1,50 @@
 .percentage-bar-chart {
-  display: flex;
-  background-color: white;
+  box-sizing: border-box;
+  background-color: var(--g3-color__white);
   padding: 12px 12px;
   font-size: 14px;
   line-height: 1em;
   letter-spacing: .03rem;
-  color: #606060;
+  color: var(--g3-color__gray);
 }
 
-.percentage-bar-chart__legend {
+.percentage-bar-chart__title-box {
```

Comment:
What's the reason these styles are duplicated in data portal and gen3-ui-component? src/components/charts/PercentageStackedBarChart/PercentageStackedBarChart.css

Reply:
Pretty sure they are there before we put them into `gen3-ui-component`. And I think only the old data explorer (powered by arranger) are still importing those.

I'll see if I can let the arranger data explorer to import from gen3-ui-component and get rid of those duplicates. But I hope we can get rid of the whole arranger thing soon. 🤦‍♂